manifests/virt-handler.yaml.in

```diff
@@ -25,6 +25,8 @@ spec:
           - "qemu:///system"
           - "--hostname-override"
           - "$(NODE_NAME)"
+          - "--driver"
```

Comment:
this looks like it is hard coded, and should instead be done via template replacement.  This might be an argument to use Helm for the deployments.